GameStore/Views/Game/GameForm.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿<script src="~/scripts/jquery-1.10.2.min.js"></script>
+<script>
+    $(document).ready(function () {
+
+        getAllGenres();
+        getAllStudios();
+
+    })
+
+    function getAllGenres() {
```

Comment:
Let's move all requests to server to a single file (e.g. some client) to make code reusable.

Reply:
Fixed